patches/bazelbuild_bazel-gazelle_aspect-gitignore.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/config/config.go b/config/config.go
+index 23b1083..1841650 100644
+--- a/config/config.go
++++ b/config/config.go
+@@ -121,6 +121,7 @@ type MappedKind struct {
+ }
+ 
+ const ASPECT_WALKSUBDIR = "__aspect:walksubdir"
++const ASPECT_GITIGNORE = "__aspect:gitignore"
+ 
+ func New() *Config {
+ 	return &Config{
+diff --git a/walk/walk.go b/walk/walk.go
+index 7814088..8ebb086 100644
+--- a/walk/walk.go
++++ b/walk/walk.go
+@@ -165,6 +165,13 @@ func visit(c *config.Config, cexts []config.Configurer, knownDirectives map[stri
+ 		return nil, false
+ 	}
+ 
++	// PATCH(gitignore) ---
```

Comment:
FYI, I left a comment in bazebuild suggesting gazelle expose an API for configuring ignore: https://github.com/bazelbuild/bazel-gazelle/issues/1166#issuecomment-2381605399 - ideally there would be no need to patch.

Reply:
Thanks @gonzojive, note I've also implemented it within gazelle and I'm trying to get it reviewed+merged: https://github.com/bazelbuild/bazel-gazelle/pull/1908